feat: launch week day 5 announcement (#6245)
supabase/supabase: www/_blog/2022-04-01-supabrew.mdx

---
title: Supabrew - Never Code Thirsty
description: A light and refreshing non-alcoholic beer for devs.
author: ant_wilson
image: launch-week-4/friday-supabrew/productShot1.jpg
thumb: launch-week-4/friday-supabrew/productShot1.png
tags:
  - launch-week
date: '2022-04-01'
toc_depth: 3
---

> Supabase is best known as being a Developer Experience focused database company. As we expand our offerings and feature set, it’s only natural that we expand into other green field areas of the Developer Experience stack. After all, Supabase exists to solve developer problems, wherever we find them.

Reviewer: ```suggestion
Supabase is best known as being a Developer Experience focused database company. As we expand our offerings and feature set, it’s only natural that we expand into other greenfield areas of the Developer Experience stack. After all, Supabase exists to solve developer problems, wherever we find them.
```

> One thing we noticed that ALL developers require when they sit down to code is refreshment. There’s nothing worse than being in the middle of a 4000 line PR review and experiencing an extreme, unquenchable thirst. After extensive market review, we discovered that there doesn’t yet exist a product to fill this gap in the market.

Reviewer: ```suggestion
One thing we noticed ALL developers require when they sit down to code is refreshment. There’s nothing worse than being in the middle of a 4000 line PR review and experiencing an extreme, unquenchable thirst. After extensive market review, we discovered that there doesn’t yet exist a product to fill this gap in the market.
```

![copplepack.jpg](/images/blog/launch-week-4/friday-supabrew/copplepack.jpg)

We surveyed our 80,000 registered developers, and asked them if they could help us design the perfect beverage for coding - what would it be? The result was a light, refreshing, low sugar, and non-alcoholic beverage.

If you’ve been following our social media recently you will have seen that the entire team is currently working from a remote location in Bavaria, Germany. This is no coincidence. We came here primarily to sample beverages from some of the world’s oldest and best breweries. And we are proud to announce for the first time today, our latest solution to Developer Experience problems has arrived...

### SupaBrew. An all-natural, light and refreshing non-alcoholic beer - engineered specifically for the needs of devs.

![antines.png](/images/blog/launch-week-4/friday-supabrew/antines.jpg)

> Brewed in accordance with strict Germany Beer Purity laws, SupaBrew is made from 3 open source ingredients: Barley, Hops, and Water. With the absence of preservatives, you can be sure that you’re feeding your brain with only the purest liquids, which will keep your mind sharp, enabling you to squash bugs and center divs late into the night, and through to the early hours of the morning. And it’s not just an evening beverage, we specifically adapted the taste so you can enjoy it during your 9am stand ups.

Reviewer: ```suggestion
Brewed in accordance with strict Germany Beer Purity laws, SupaBrew is made from 3 open source ingredients: Barley, Hops, and Water. With the absence of preservatives, you can be sure that you’re feeding your brain with only the purest liquids, which will keep your mind sharp, enabling you to squash bugs and center divs late into the night, and through to the early hours of the morning. And it’s not just an evening beverage, we specifically adapted the taste so you can even enjoy it during your 9am stand ups.
```
